nvtabular/inference/triton/__init__.py

```diff
@@ -463,6 +472,13 @@ def get_column_types(path):
     return json.load(open(os.path.join(path, "column_types.json")))
 
 
+def get_slot_sizes(path):
+    if not path.exists(path):
+        return json.load(open(os.path.join(path, "slot_size_array.json")))
```

Comment:
Why are you checking if the path doesn't exist? Also if the path doesn't exist, how can the slot_size_array.json file inside the path exist?

Reply:
Updated it.